swarm.cabal

```diff
@@ -191,6 +191,7 @@ library swarm-lang
     terminal-size >=0.3 && <1.0,
     text,
     text-rope >=0.2 && <0.3,
+    unicode-show,
```

Comment:
Since this is the only place we have this listed, let's include bounds, like `unicode-show >= 0.1 && <0.2`.